contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -816,6 +825,29 @@ public void runPushStore(Props props, String url, String dataDir) throws Excepti
 
         log.info("Push starting for cluster: " + url);
 
+        // CDN distcp
+        boolean cdnEnabled = props.getBoolean(PUSH_CDN_ENABLED, false);
+        String modifiedDataDir = new Path(dataDir).makeQualified(FileSystem.get(new JobConf())).toString();
+        String storeWhitelist = props.getString(PUSH_CDN_STORE_WHITELIST, null);
+        GobblinDistcpJob distcpJob = null;
+
+        if (cdnEnabled && storeWhitelist != null && storeWhitelist.contains(storeName)) {
```

Comment:
@cqgao : FYI, `VoldemortProperties` have a `getList` function that should do what you need.